examples/Components/topology/cube5x5x5-surface.obj

```diff
@@ -0,0 +1,951 @@
+g 1_Visual
```

Comment:
To be removed or moved to share/mesh if needed by a scene.

Reply:
Result of the export of ```examples/Components/topology/Hexa2TetraTopologicalMapping_export.scn```, removed